configurecompiler.cmake

```diff
@@ -477,6 +477,8 @@ if (CLR_CMAKE_PLATFORM_UNIX)
   else()
     add_compile_options(-Wno-unused-variable)
     add_compile_options(-Wno-unused-but-set-variable)
+    set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wno-discarded-qualifiers")
```

Comment:
Should this rather be added in `src/corefx/System.Globalization.Native/CMakeLists.txt` where the warning is disabled for Clang?

Reply:
done